src/components/Students.jsx

```diff
@@ -0,0 +1,75 @@
+import {useState, useEffect} from 'react';
+import {Container, Row, Card} from 'react-bootstrap';
+import {useHistory} from 'react-router-dom';
+
+const Students = () => {
+    let history = useHistory();
+    let [students, setStudents] = useState(null);
+    let [loading, setLoading] = useState(true);
+    
+    useEffect(() => {
+        setLoading(true);
+
+        fetch('url')
+        .then(response => response.json())
+        .then(data => {
+            setLoading(false);
+            setStudents(data);
+        })
+    },[])
+
+    if(!loading) {
+        if(students != null) {
```

Comment:
similar to courses, I would do an early return for the `loading` state and I would set the initial state of `student` as an empty array